src/server/stage-3eme/infra/repositories/apiImmersionFacileStage3eme.repository.test.ts

```diff
@@ -30,6 +30,7 @@ describe('ApiImmersionFacileStage3emeRepository', () => {
 			// Then
 			expect(httpClientService.get).toHaveBeenCalledWith(expect.stringContaining('/search'));
 		});
+
 		it('appelle l’api Immersion Facile avec la localisation fournis', () => {
 			// Given
 			const filtre: Stage3emeFiltre = aStage3emeFiltre();
```

Comment:
```suggestion
			const filtre: Stage3emeFiltre = aStage3emeFiltre({
			  		distanceCommune: '10',
		                         latitudeCommune: '2',
		                         longitudeCommune: '3',
			});
```

C'est "pareil" mais ça permet d'avoir toutes les informations dans le test / de changer la fixture sans casser le test